Dockerfile

```diff
@@ -33,7 +33,7 @@
 # Stage 1 - Acquire the CARMA source as well as any extra packages
 # /////////////////////////////////////////////////////////////////////////////
 
-FROM usdotfhwastol/carma-base:2.8.3 AS source-code
+FROM usdotfhwastol/autoware.ai:latest AS source-code
```

Comment:
Not sure how to feel about committing this as `latest', but since we don't yet have the images for autoware built not sure what to do. If the dockerization PR is merged in the autoware repo we can merge that to master, tag it with the component version, and publish it to dockerhub then reference that here. Might be better to sequence that way.

Reply:
I am not opposed to changing this. What version number would be best?

Reply:
`usdotfhwastol/autoware.ai:3.0.0` is the new one just published.